README.md

```diff
@@ -107,21 +107,57 @@ prob = ODEProblem(f,u0,tspan,p)
 sol = solve(prob,Tsit5())
 ```
 
-## Differences from NamedTuples
+## Relation to NamedTuples
```

Comment:
I think we should also mention the NamedTuple constructors above when showing how to use the package.